spark-jobs/src/test/resources/data/standardization_csv_suite_expected_no_null_value.txt

```diff
@@ -3,7 +3,7 @@
 +-----------+----------+---------+----------------------------------------------------------------------------------------------------------------+
 |a          |Arya      |Stark    |[]                                                                                                              |
 |b          |Jon       |Snow     |[]                                                                                                              |
-|c,Use,Comma|null      |         |[[stdNullError, E00002, Standardization Error - Null detected in non-nullable attribute, last_name, [null], []]]|
+|c,Use,Comma|null      |         |[{stdNullError, E00002, Standardization Error - Null detected in non-nullable attribute, last_name, [null], []}]|
```

Comment:
**Suggestion to discuss:** It seems having these exact text output comparisons in tests is not feasible. Let's switch to binary comparison

Reply:
#1772

Reply:
Done.